### DISCUSSION TITLE
Not able to run integration with harmony on scATACseq datasets

### DISCUSSION THREAD

**Discussion by pobmmu** [stuart-lab/signac#1632] [Q&A]
Hello, 

I am having trouble to integrate two scATACseq datasets with Harmony. I get this error message: 

merged.harmony <- RunHarmony(merged, group.by.vars = "dataset", reduction = "lsi", assay.use = "ATAC", project.dim = FALSE)
Error in RunHarmony.Seurat(merged, group.by.vars = "dataset", reduction = "lsi",  : 
  argument 3 matches multiple formal arguments

I have created a common peak set before merging the datasets with the merge() function.

**Comment by timoast** [comment]
The `RunHarmony()` function is part of the harmony package, please raise an issue there: https://github.com/immunogenomics/harmony
